parlai/core/build_data.py

```diff
@@ -383,7 +383,7 @@ def download_models(
                     url += model_type + '/'
                 url += fname
             else:
-                url = path + '/' + fname
+                url = path + '/' + model_folder + '/' + fname
```

Comment:
hmmm... this worries me. will this break anything?

Reply:
ah, I can get rid of this. I was using this during localhost testing (if you follow the quip we have, you don't end up wget-ing the right file)

Reply:
drop this still tho